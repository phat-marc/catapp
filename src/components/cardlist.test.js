import { shallow } from 'enzyme';
import React from 'react';
import CardList from './cardlist';
// import renderer from 'react-test-renderer';

it('expect to render cardlist component', () => {
	const mockCatFriends = [
		{
			id: 1,
			name: 'wiggles',
			email: 'wiggle@kitty.com'
		}
	];
	expect(shallow(<CardList catfriends={mockCatFriends}/>)).toMatchSnapshot();
})
// Here we need to pass the mock cats for the test to work correctly