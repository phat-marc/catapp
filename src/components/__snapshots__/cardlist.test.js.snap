// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardList expect to render cardlist component 1`] = `ShallowWrapper {}`;
